app/assets/javascripts/oxalis/view/input_catchers.js

```diff
@@ -34,10 +37,30 @@ class InputCatchers extends React.PureComponent<Props, State> {
       width: width / 4 - 0.5,
     };
 
+    const { brushPosition, brushSize } = this.props;
+    const brush =
+      brushPosition != null ? (
+        <div
+          id="cursor"
+          style={{
+            position: "relative",
+            left: brushPosition[0] - brushSize / 2,
+            top: brushPosition[1] - brushSize / 2,
+            width: brushSize,
```

Comment:
Maybe move all attributes (except left/top) into a css class?

Reply:
Perhaps add the will-change CSS attribute for max perf? https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Reply:
We agreed to keep these CSS properties there for now. Regarding will-change, as the mdn site specifically mentions to refrain from premature optimization as it causes overly complicate rendering and I do not experience performance issues while moving the cursor, I would not add the property for now.